src/www/views/servers_view/server_connection_indicator/index.ts

```diff
@@ -174,7 +170,7 @@ export class ServerConnectionIndicator extends LitElement {
       ${CIRCLE_SIZES.map(
         circleSize =>
           html`
-            <img class="circle circle-${circleSize} circle-${this.animationState}" src="${circle}" />
+            <img class="circle circle-${circleSize} circle-${this.animationState}" src="${circle}" height="100%" />
```

Comment:
Just curious, what's the difference between this inline attribute and `height: 100%` in `.circle` css style?

Reply:
This resizes the actual image itself, using the image data to ensure square proportion. Not entirely recommended, as it couples the layout to the image data. But I doubt we plan on changing the image?